backend/app/usecases/chat.py

```diff
@@ -255,6 +264,55 @@ def insert_knowledge(
     return conversation_with_context
 
 
+def get_bedrock_response(args: dict):
+
+    client = get_bedrock_client()
+    messages = args["messages"]
+
+    logger.debug(f"model: {args['model']}")
+
+    prompt = "\n".join(
+        [
+            message["content"][0]["text"]
+            for message in messages
+            if message["content"][0]["type"] == "text"
+        ]
+    )
+
+    if is_mistral_model(args["model"]):
+        prompt = f"<s>[INST] {prompt} [/INST]"
+
+    logger.debug(f"Final Prompt: {prompt}")
```

Comment:
Please remove all debug logs. Or describe clearly.

example: `logger.debug(args)`is not enough.